ksqldb-rocksdb-config-setter/src/test/java/io/confluent/ksql/rocksdb/KsqlBoundedMemoryRocksDBConfigSetterTest.java

```diff
@@ -172,6 +189,10 @@ public void shouldSetConfig() {
     verify(rocksOptions).setWriteBufferManager(bufferManager);
     verify(rocksOptions).setStatsDumpPeriodSec(0);
     verify(rocksOptions).setTableFormatConfig(tableConfig);
+    verify(rocksOptions).setCompactionStyle(COMPACTION_STYLE);
+    verify(rocksOptions).setCompressionType(COMPRESSION_TYPE);
+    verify(rocksOptions).setMaxBackgroundJobs(MAX_BACKGROUND_JOBS);
+    verify(rocksOptions.compactionOptionsUniversal()).setAllowTrivialMove(ALLOW_TRIVIAL_MOVE);
```

Comment:
Can we add a separate test for compaction style of "LEVEL," since the behavior of this line will be different? For level compaction, we can verify that `rocksOptions.compactionOptionsUniversal()` is never called.